doc/imported_samples/marking_criteria.yml

```diff
@@ -0,0 +1,36 @@
+Quality of Writing:
+    max_mark: 24.0
+    level_0:
+        name: Beginner
+        description: The essay is very poorly organized structure and gives no new information.
+    level_1:
+        name: Capable
+        description: The essay is poorly organized but gives new information.
+    level_2:
+        name: Accomplished
+        description: The essay is well-structure and conveys new information clearly.    
+    ta_visible: False
```

Comment:
For the first entry, show default values (`ta_visible` is true and `peer_visible` is false by default.